src/main/java/com/softserve/edu/sporthubujp/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,12 @@
+package com.softserve.edu.sporthubujp.mapper;
+
+import com.softserve.edu.sporthubujp.dto.CategoryDTO;
+import com.softserve.edu.sporthubujp.entity.Category;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+@Mapper
+public interface CategoryMapper {
+    CategoryMapper INSTANCE = Mappers.getMapper(CategoryMapper.class);
```

Comment:
you can just put @Mapper(componentModel = "spring", builder = @Builder(disableBuilder = true))
and inject mappers as Spring Beans